specification/attestation.adoc

```diff
@@ -1,309 +1,880 @@
-|===
+Since they hold TCB elements measurements bound to the platform lifetime, the
+platform/TSM measurement registers may be stored in the hardware Root of Trust
+for Measurement (RTM). The TVM measurements must be maintained in TSM
+confidential memory with TVM bound life cycle, effectively making the TSM an
+extended RTM for the TVM.
 
-This extension defines attestation evidence about the DICE layer that is 
-associated with the Subject key. The certificate Subject and 
-SubjectPublicKey identify the entity to which the DiceTcbInfo extension 
-applies. When this extension is used, the measurements in the evidence 
-usually describe the software/firmware (and configuration) which will 
-execute within the TCB. The AuthorityKeyIdentifier extension <<R2>> MUST be 
-supplied when the DiceTcbInfo extension is supplied. This allows the 
-Verifier to locate the signer’s certificate. The DiceTcbInfo extension 
-should be included with CRL entries that revoke the certificate that 
-originally included the said DiceTcbInfo extension. 
-
-For TVM attestation, the following TCB Evidence Info will be sequenced 
-using the above DiceTcbInfo structure. Multiple evidences may be provided 
-via the *MultiDiceTcbInfo* extension:
-
-* Cryptographic hash of the RoT FW binary and configuration, along with its 
-SVN and other fields; 
-* Cryptographic hash of the TSM-driver binary and configuration, along with 
-its SVN and other fields ; 
-* Cryptographic hash of the TSM binary and configuration, with its SVN and 
-other fields; 
-* Cryptographic hash of the OSAM (described below) binary and 
-configuration, with its SVN and other fields - this is applicable for 
-remote attestation only;
-  ** If OSAM is a 3rd party - the certifying entity will need a separate 
-evidence entry. 
-* Cryptographic hash of the TVM static binaries and configuration, along 
-with its SVN and other fields.
-* The TVM may additionally extend cryptographic measurements for other 
-workload binaries and configuration loaded dynamically subsequent to boot 
-via the TG-ABI.
-
-The TVM TCB Evidence Info is managed by the TSM and is combined with the 
-TSM’s TCB Evidence info that is in turn managed by the TSM-driver. The 
-TSM-driver provides a TEEI security routine to enable the TSM and 
-transitively the TVM to generate an Attestation CDI (Composite Device 
-Identifier) and key to participate in an Attestation certificate-based 
-protocol for remote (and local) attestation.
-
-We recommend at least the following CDIs to be supported for AP-TEE 
-workloads:
-
-. Attestation CDI - This CDI is derived from the combination of the input 
-values listed above and is expected to change across software updates or 
-configuration changes of these components. This CDI is meant for remote 
-attestation and is mandatory for AP-TEE implementations.
-. Versioned Sealing CDI - This CDI is also derived from the combination of 
```

Comment:
There is no specification of versioned sealing CDI in this new attestation specification.

Reply:
Good point, I will add it as a separate commit.

Reply:
ok